rlclientlib/azure_factories.cc

```diff
@@ -31,13 +31,34 @@ namespace reinforcement_learning {
   }
 
   int restapi_data_transport_create(m::i_data_transport** retval, const u::configuration& config, i_trace* trace_logger, api_status* status) {
-    const auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
+    auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
+    auto usingAPI = false;
+
     if (uri == nullptr) {
-      RETURN_ERROR(trace_logger, status, http_uri_not_provided);
+      uri = config.get(name::MODEL_HTTP_APPI_HOST, nullptr);
```

Comment:
same about naming here

Reply:
renamed it to model_uri as it can be either blob uri or endpoint